src/pages/List.js

```diff
@@ -14,6 +14,7 @@ const List = () => {
         path="items"
         // Sort the data
         sort="name"
+        // filter={['token', '==', 'baron flew gibby']}
         // isLoading = is a Boolean that represents the loading status for the firebase query. true until an initial payload from Firestore is received.
```

Comment:
@ajiles91  I know what the problem is. This is creating a compound query (order by name, then filter by token) and an index needs to be created for it to work.

I'm on it now. Nothing was wrong with our syntax, it's a Firebase thing.

Reply:
Also stackoverflow to the rescue - https://stackoverflow.com/questions/58261612/firestore-orderby-and-where-filters-not-working-together

Reply:
@segdeha @ajiles91 We manually added the token to these items in the database and displayed them using the hard-coded token for the query. Does this satisfy the story? 
![List associated with token](https://p112.p2.n0.cdn.getcloudapp.com/items/Jru7DD62/Image+2020-01-01+at+6.49.46+PM.png?v=cc1ec73491123be99143509cd884c8e7)

Reply:
This is on the right track! What would be better in terms of completing this in a way that let's us continue to build out the app is to read the token from `localStorage` and filter using that. So, to test this, you could manually set the token in `localStorage` and make sure your list pulls the right items. Let me know if you have any questions about that!

![image](https://user-images.githubusercontent.com/4306/71650333-f7765f00-2cc9-11ea-9151-70285743ec6e.png)

Reply:
That makes perfect sense to me. Thanks Andrew. cc @ajiles91